week-1/schedule.md

```diff
@@ -0,0 +1,68 @@
+| 11:00   | [Community talk](https://facresources.com/slides/community-talk#/)                                                                                                                                                                                                                                                                      |                   |
+| _11:30_ | Go through [course schedule](https://github.com/foundersandcoders/master-reference/tree/master/coursebook), [code of conduct](https://github.com/foundersandcoders/master-reference/blob/master/code-of-conduct.md), [house rules](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/general/house-rules.md) |                   |
+| 12:00   | [Students create their own code of conduct](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/week-1/cohort-code-of-conduct.md)                                                                                                                                                                              |
+| 12:45   | Introduce Slack channels                                                                                                                                                                                                                                                                                                                |                   |
+| 13:00   | _Lunch_                                                                                                                                                                                                                                                                                                                                 |                   |
+| 14:00   | [Intro to Pair Programming](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/week-1/pair-programming.md)                                                                                                                                                                                                    | Pair programming  |
+| 15:00   | [Accessibility workshop](https://github.com/foundersandcoders/web-accessibility/blob/master/putting-yourself-in-someone-elses-shoes.md)                                                                                                                                                                                                 | Accessibility     |
+| 16:00   | GitHub scavenger hunt + [Kahoots quiz](https://create.kahoot.it/share/github-scavenger-hunt/f3992f24-666f-40cb-9ac5-56ede85e202d)                                                                                                                                                                                                       |                   |
```

Comment:
Feedback from curriculum circle 07/02: swap this for estimation/user story material